specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2022-03-01/checkPolicyRestrictions.json

```diff
@@ -0,0 +1,408 @@
+        },
+        "x-ms-examples": {
+          "Check policy restrictions at resource group scope": {
+            "$ref": "./examples/PolicyRestrictions_CheckAtResourceGroupScope.json"
+          }
+        }
+      }
+    },
+    "/providers/{managementGroupsNamespace}/managementGroups/{managementGroupId}/providers/Microsoft.PolicyInsights/checkPolicyRestrictions": {
+      "post": {
+        "operationId": "PolicyRestrictions_CheckAtManagementGroupScope",
+        "description": "Checks what restrictions Azure Policy will place on resources within a management group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/managementGroupsNamespaceParameter"
+          },
+          {
+            "$ref": "#/parameters/managementGroupIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CheckManagementGroupRestrictionsRequest"
+            },
+            "description": "The check policy restrictions parameters."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The restrictions that will be placed on the resource by Azure Policy.",
+            "schema": {
+              "$ref": "#/definitions/CheckRestrictionsResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../stable/2019-10-01/policyMetadata.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Check policy restrictions at subscription scope": {
+            "$ref": "./examples/PolicyRestrictions_CheckAtManagementGroupScope.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CheckRestrictionsRequest": {
+      "description": "The check policy restrictions parameters describing the resource that is being evaluated.",
+      "properties": {
+        "resourceDetails": {
+          "description": "The information about the resource that will be evaluated.",
+          "$ref": "#/definitions/CheckRestrictionsResourceDetails"
+        },
+        "pendingFields": {
+          "description": "The list of fields and values that should be evaluated for potential restrictions.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PendingField"
+          }
+        }
+      },
+      "required": [
+        "resourceDetails"
+      ]
+    },
+    "CheckManagementGroupRestrictionsRequest": {
+      "type": "object",
+      "description": "The check policy restrictions parameters describing the resource that is being evaluated.",
+      "properties": {
+        "resourceDetails": {
```

Comment:
>resourceDetails [](http://example.com/codeflow?start=9&length=15)

I thought they can't pass any resource details

Reply:
Yes, I wanted to allow for future support for resource content and to keep the contract as similar as possible to the subscription-level API.
I don't think we'll be supporting resource content any time soon though, so also I'm fine with removing it if you prefer.